routes/truckRoutes.js

```diff
@@ -1 +1,41 @@
-// put your Express router code in here
\ No newline at end of file
+var express = require('express');
+//var trucks = require('../foodTrucks');
+var router = express.Router();
+
+
+
+router.route('./trucks')
+  .get(function (request, response) {
+    var truckList = trucks.getTrucks();
+    response.send(truckList);
+  })
+  .post(function (request, response) {
+    var truckList = trucks.addTruck;
+    response.send(truckList);
+  });
+
+router.route('./trucks/:name')
+  .get(function (request, response) {
+    var truck = trucks.getTruck(request.params.name);
+    response.send(truck);
+  })
+  .delete(function (request, response) {
+    var truckList = trucks.removeTruck();
+    response.send(truckList);
+  });
+
+router.route('/food-types')
```

Comment:
These routes aren't part of the `/trucks` path - either leave these route handlers in index.js, or create a foodTypesRouter module and move them in there.